pkg/liquidity_provider.go

```diff
@@ -59,8 +60,8 @@ type PegoutConfigurationDTO struct {
 	TimeForDeposit       uint32  `json:"timeForDeposit" validate:"required"`
 	ExpireTime           uint32  `json:"expireTime" validate:"required"`
 	PenaltyFee           string  `json:"penaltyFee" validate:"required,numeric,positive_string"`
-	FixedFee             string  `json:"fixedFee" validate:"required,numeric,positive_string"`
-	FeePercentage        float64 `json:"feePercentage" validate:"required,numeric,gt=0"`
+	FixedFee             string  `json:"fixedFee" validate:"required,numeric,min=0"`
```

Comment:
```suggestion
	FixedFee             string  `json:"fixedFee" validate:"required,numeric,positive_string"`
```

Reply:
in this case you can use `positive_string ` that validator is the same as min=0 but for string fields, the reason why this field and the others that are amounts are strings is because the value in them might be bigger than a `uint64`